medicai/layers/attention.py

```
@@ -161,3 +163,213 @@ def get_config(self):
+        self.qkv_bias = qkv_bias
+        self.channel_attn_drop = channel_attn_drop
+        self.spatial_attn_drop = spatial_attn_drop
+        self.prefix = name
+
+        if hidden_size % num_heads != 0:
+            raise ValueError(
+                f"hidden_size ({hidden_size}) must be divisible by num_heads ({num_heads})"
+            )
+        self.head_dim = hidden_size // num_heads
+
+    def build(self, input_shape):
```

Ruff (0.14.8)

Unused method argument: `input_shape`

(ARG002)